node-nut.js

```diff
@@ -57,6 +64,7 @@ Nut.prototype.close = function () {
 	this._client.end();
 };
 
+var vars = [];
```

Comment:
to define the variable scope explizit

Reply:
I'm not a JS programmer, but... are you sure `vars` should be declared in this scope? Would not be better to declare it inside `parseKeyValueList()`?

Excuse me, as I said, my knowledge of JS is quite limited and I can't test it now because I don't have an UPS in this house.

Reply:
since `parseKeyValueList` will be called multible times for each TCP packet here:
https://github.com/Links2004/node-nut/blob/9e25b557e46804715122c448e33f299bb2e4f4a9/node-nut.js#L38
via here (for example)
https://github.com/Links2004/node-nut/blob/9e25b557e46804715122c448e33f299bb2e4f4a9/node-nut.js#L94
the data need to be stored outside the function or it is lost.